utils/helper.ts

```diff
@@ -21,4 +21,37 @@ const validationSchema = {
 		.oneOf([yup.ref('password'), null], 'Passwords do not match'),
 }
 
-export default validationSchema
+export const editProfileValidationSchema = {
```

Comment:
This is repetition, you can add this to the first validation schema, there is no need to have multiple validation schema. add the wt, height, bio e.t.c to the validationSchema object, no need for duplicates it will work just fine

One the last name schema, the required message should read 'Last name is required" not "First name is required"